tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -367,6 +367,12 @@ def create_parser(product_choices=None):
         action="store_true",
         dest="sanitizer_enabled",
         help="Only alert on sanitizer-related errors and crashes.")
+    chrome_group.add_argument(
+        "--reuse-window",
```

Comment:
Do we plan to use wptrunner with both flag on and off? If we always plan to pass in this flag, we may not need another command line argument?

Reply:
Ack